Formula/rpm.rb

```diff
@@ -34,6 +34,13 @@ class Rpm < Formula
   depends_on "gettext"
   depends_on "xz"
   depends_on "ossp-uuid"
+  # use external libraries, until the internal have been fixed
+  depends_on "lua"
+  depends_on "syck"
+  depends_on "openssl"
+
+  # lots of OS X support things were missing from this release
+  patch :DATA
```

Comment:
OK, I understand that goal. Meanwhile, afb is essentially "upstream" (he's part of the RPM5 project), he always sends patches and careful bug reports, and I can/will release a tarball for almost any need. What may not be obvious to you is that RPM can be configured in so many ways (my devel build links >100 libraries, essentially what I release) and with a configuration space of that magnitude, there are almost always some surprises when disabling functionality. The main problem with rpm-5.4.16 was an accidental name space collision on HFS between "rpmhash.c" and "rpmhash.C"